release/RELEASE.md

```diff
@@ -113,34 +113,51 @@ This file lists all the steps to follow when releasing a new version of ODS Andr
 
 - Click `Integrations`, find the uploaded bundle in the `Artifacts` sections and click `Create release`.
 
-    ![Create OMA Portal release](images/google_play_store_release_01.png) 
+  ![Create OMA Portal release](images/google_play_store_release_01.png)
 
 - Progress through the various submission steps.
 
 - When step is `Store submissions`, select `Closed beta with Google Groups`, set `Email addresses` to `odsapp@googlegroups.com` and click `Go for store submission`.
 
-    ![OMA Portal store submission](images/google_play_store_release_02.png)<br /><br />
+  ![OMA Portal store submission](images/google_play_store_release_02.png)<br /><br />
+
+- To publish the release you have to click a last time on "Got to publication". You should receive a confirmation email from OMA with a "Published" status.
+
+### Announce the new release on GitHub & FoODS
+
+- Go to [GitHub ods-android - Discussions > Announcements](https://github.com/Orange-OpenSource/ods-android/discussions/categories/announcements)
+
+- Click to `New discussion`
+
+- Set title to `ODS Android library X.Y.Z`
+
+- Post a message with screenshots of new elements to announce the release
+
+Here is an example of the v0.15.0 release announcement:
+
+```markdown
+**ODS Android library v0.15.0 is here!**
+Be careful, this version includes several APIs change in the library in order to prepare the v1.0 release.
+
+### New components
+
+**- List item
+**: It allows you to display a Checkbox, a Switch, a RadioButton, an Icon or a Caption text as trailing element. If this does not meet your needs, you can use another signature which accept any Composable as trailing.
+
+<img width="441" alt="Capture d’écran 2023-09-13 à 15 06 07" src="https://github.com/Orange-OpenSource/ods-android/assets/17410558/7ff8f0e4-1dca-4f5b-9ce8-ee0b9f86cb1d">
+
+<img width="442" alt="Capture d’écran 2023-09-13 à 15 06 32" src="https://github.com/Orange-OpenSource/ods-android/assets/17410558/17c6bed1-b565-4140-abf8-e1e7107cf59a">
+<br/><br/>
+
+**- Toggle text buttons group**:
+<img width="443" alt="Capture d’écran 2023-09-13 à 15 03 34" src="https://github.com/Orange-OpenSource/ods-android/assets/17410558/95428f63-5ddd-457f-a9e7-c89d38d5e85f">
+
+---
+🗒️ Please review the [v0.15.0 release note](https://github.com/Orange-OpenSource/ods-android/releases/tag/0.15.0) for a complete list of changes since our previous release.
 
-### Announce the new release on FoODS
+📱 You can subscribe to the [Orange Design System Google group](https://groups.google.com/g/odsapp) in order to be able to install the [Demo app](https://play.google.com/apps/testing/com.orange.ods.app).
+```
 
 - Go to [Teams - FoODS: ODS Mobile Android](https://teams.microsoft.com/l/channel/19%3a62efff7aca9f4865a841076b4a6c428f%40thread.tacv2/4-ODS%2520Mobile%2520Android?groupId=a9581e9e-6775-46f5-9e4a-3ce57096e6be&tenantId=90c7a20a-f34b-40bf-bc48-b9253b6f5d20)
```

Comment:
As discussed during the daily, we should remove the FoODS Teams URL. I know that you probably did not have time to do it yet but I write it down so that we don't forget this one 😄